SeeSharpSnake.csproj

```diff
@@ -46,5 +52,39 @@
     <IlcDisableReflection>true</IlcDisableReflection>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(Mode)' == 'CoreRT-CustomPal'">
+    <NoStdLib>true</NoStdLib>
+    <NoConfig>true</NoConfig>
+    <RuntimeMetadataVersion>v4.0.30319</RuntimeMetadataVersion>
+    <Optimize>true</Optimize>
+    <IlcOptimizationPreference>Size</IlcOptimizationPreference>
+    <IlcDisableReflection>true</IlcDisableReflection>
+    <IncludePal>true</IncludePal>
+    <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
+    <GenerateTargetFrameworkAttribute>false</GenerateTargetFrameworkAttribute>
+  </PropertyGroup>
+
+  <ItemGroup Condition="'$(Mode)' == 'CoreRT-CustomPal'">
+    <IlcArg Include="--systemmodule:SeeSharpSnake" />
```

Comment:
Could you submit a pull request to CoreRT repo to expose the system module argument as a property? (Could be e.g. `IlcSystemModule`.)

Then we can also condition `AutoInitializedAssemblies` on `IlcSystemModule` not being set (i.e. we can make it so that the targets won't add any AutoInitializedAssemblies if IlcSystemModule is set, removing the need to `Remove` them here. If someone is running with a custom system module, they're likely not compatible with the rest of the modules either.)

Reply:
Ok. I will add `IlcSystemModule` as property and I more then agree with make property which will disable generation of AutoInitializeAssemblies. Given that you satisfied with such approach I will take this as baseline of my understanding and move forward according to your comments. Don’t forget to look at ZeroSharp project :)